pkg/unvanquished_src.dpkdir/ui/transition_downloading.rml

```diff
@@ -5,7 +5,7 @@
 		<link type="text/rcss" href="loading.rcss" />
 		<style>
 			body {
-				background-color: black;
+				background-color: #00000000;
```

Comment:
I think the number of digits is supposed to be divisible by 3? Like `#000` or `#000000`.

Reply:
Oh apparently 8 digits means there is an alpha component. So that should make a completely invisible color with alpha = 0!

Reply:
Yep, it's a hex rgba.

Yes, making it invisible is what I went for. For whatever reason, the stuff under `body id="download"` gets rendered before the `head` part. I tried changing the `z-index` there, but it did not seem to work.

Reply:
What do you mean "the head part"?

I don't understand how making the background invisible is supposed to work. Would we see whatever window is drawn below the download one?

Reply:
Everything between `<head></head>`.

Right now we'd see the connection screen under it. That can be changed to just draw a black background though.

Reply:
> Everything between `<head></head>`.

That's just style directives. There isn't any content.